composer.json

```diff
@@ -54,6 +54,7 @@
         "drupal/twig_tweak": "^3.1",
         "drush/drush": "^10.3.5",
         "geocoder-php/nominatim-provider": "^5.2",
+        "knplabs/github-api": "^3.0",
```

Comment:
@gkapoor121212, can we do without this line? We already have this in the specific composer.json inside `ct_github`.